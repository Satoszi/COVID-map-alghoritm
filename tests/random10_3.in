10 10 4 28
random10_3
+.|+..|...
-..+|.++|.
.....-.++.
+-.-++.+.-
.+.+++++++
+..+.-+.+-
.|-+..++..
..+..+|+||
+...-|++..
||+.+.++.+
