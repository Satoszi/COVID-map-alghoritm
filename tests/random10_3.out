9 9
4 9
2 9
1 9
7 8
4 8
0 8
9 7
8 7
5 7
2 7
2 6
1 6
9 5
5 5
0 5
1 4
9 3
3 3
0 3
5 2
8 1
4 1
0 1
6 0
3 0
2 0
0 0
